helmfile.d/charts/prometheus-alerts/templates/alerts/cluster-capacity-management-alerts.yaml

```
@@ -137,4 +137,13 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: NodeMissingElastisysLabel
+      annotations:
+        description: "The node {{`{{`}} $labels.node {{`}}`}} in {{`{{`}} $labels.cluster {{`}}`}}  is missing the 'elastisys.io/node-group' label. This might affect capacity management."
```

Maybe make it clearer here that this affects capacity management _monitoring_:
```suggestion
        description: "The node {{`{{`}} $labels.node {{`}}`}} in {{`{{`}} $labels.cluster {{`}}`}}  is missing the 'elastisys.io/node-group' label. This might affect capacity management monitoring."
```